### ISSUE TITLE
small typo setting stroke property

### ISSUE THREAD

**Issue by SneakyWhoami** [MattMcFarland/phaser-menu-system#6]
Copy-pasted code sometimes sets a stroke value to `"rgba(0,0,0,0"` or similar (bracket doesn't close). Trivial little typo

**Comment by MattMcFarland** [comment]
Can you send a PR for the issues you fixed?

**Comment by SneakyWhoami** [comment]
Yep! Once I finish the little toy I was working on I'll send through some PR. Should be by the start of next week. Thanks for this, by the way, I've learned heaps from it!
